x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -1221,6 +1222,31 @@ private static void renderFullLayout(String name, boolean hasExamples) throws IO
         writeToTempDir("layout", rendered, "asciidoc");
     }
 
+    private static void renderKibanaInlineDocs(String name, FunctionInfo info) throws IOException {
+        StringBuilder builder = new StringBuilder();
+        builder.append("""
+            <!--
+            This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+            -->
+
+            """);
+        builder.append("### ").append(name.toUpperCase(Locale.ROOT)).append("\n");
+        builder.append(info.description()).append("\n\n");
+
+        if (info.examples().length > 0) {
+            Example example = info.examples()[0];
```

Comment:
It would be nice to have all examples. But ok for the first one as first step.

Reply:
I'm happy to include all of them if you want. I just didn't see a place where you had more than one.